recipes/dhall-python/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "dhall-python" %}
+{% set version = "0.1.14" %}
+{% set sha256 = "d6b95f03d25c80a39267891b6401be6af411e5eac44591a1654267a96533c06c" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/s-zeng/dhall-python/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  script:
+    - cargo-bundle-licenses --format yaml --output THIRDPARTY.yml
+    - maturin build --release -i {{ PYTHON }}
+    - {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('rust') }}
+    - cargo-bundle-licenses
+    - maturin >=0.14,<0.15
+    - pip
+    - make
+    - openssl                # [win]
+    - perl                   # [win]
+  host:
+    - python
+    - pip
+    - maturin >=0.14,<0.15
+    - openssl                # [win]
+  run:
+    - python
+    - libgcc-ng              # [linux]
+    - libgcc                 # [osx]
+    - m2w64-gcc              # [win]
```

Comment:
Instead of adding these libs directly, add {{ compiler('c') }} to the build section?

Or depend on `m2w64-gcc-libs` instead of the compiler package. I assume these libs are listed here because your binary needs to make system calls via libc?